external/runtime/runtime.depproj

```diff
@@ -73,6 +75,43 @@
     <Exec Command="chmod +x $(TestHostRootPath)dotnet" Condition="'$(RunningOnUnix)' == 'true'"/>
   </Target>
 
+  <PropertyGroup>
+    <_CoreCLRPackageId>runtime.$(NugetRuntimeIdentifier).Microsoft.NETCore.Runtime.CoreCLR</_CoreCLRPackageId>
+    <_CoreCLRSymbolPackageUrl>https://dotnet.myget.org/F/dotnet-core/symbols/$(_CoreCLRPackageId)/$(CoreClrPackageVersion)</_CoreCLRSymbolPackageUrl>
+    <_CoreCLRSymbolPackageDestinationDirectory>$(IntermediateOutputPath)/symbolPackages/</_CoreCLRSymbolPackageDestinationDirectory>
```

Comment:
@weshaggard do you think this should instead be the `$(PackagesDir)`? I think first it would be consistent since that is where we restore the other packages and secondly we might have to consider shortening the name for long path concerns. For example, `sym` instead of `symbolPackages` or something like that.

Reply:
The packages directory in general should only be written to by nuget restore but I could live with a cache called symbolpackages going under there as well just be sure to not put anything into the directories of the extracted packages.